source/style-guide/nested-components.txt

```diff
@@ -0,0 +1,379 @@
+.. _nested-components:
+
+============================
+Nested Components Guidelines
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. default-domain:: mongodb
+
+Use the following guidance to fix nested component issues. As you 
+fix nested component issues, you may make copy edits to existing text 
+where appropriate.
+
+If the content you edit doesn't conform well to the patterns outlined 
+in this guidance, or you have questions, pause before you proceed with 
+the changes, raise the issue in the #docs-nested-components channel on 
+Slack, and tag Sarah Simpers.
+
+For changes in versioned repos, backport the fix to all supported 
+versions.
```

Comment:
We can take this out and replace it with a more generic intro (sorry, should have been more explicit when asking you to use the info in the Google doc); this was relevant for fixing the existing issues, but they'll soon all be fixed and this guidance will just be for how to create new ones. Suggest something like this (feel free to polish as you see fit!)
```suggestion
To allow LLMs to consume our content easily, don't nest admonitions inside admonitions, admonition inside tables, or examples inside admonitions. Use the following guidance to fix existing published nested components or replace nested components in your pull requests.
```